packages/utils/crates/utils-db/Cargo.toml

```diff
@@ -14,31 +14,30 @@ async-trait = "0.1"
 async-lock = "2.7.0"
 bincode = "1.3.3"
 futures-lite = "1.12.0"
+getrandom = { workspace = true, optional = true }
 gloo-timers = { workspace = true, optional = true}
 hex = "0.4.3"
+hex-literal = "0.4.1"
 js-sys = { workspace = true, optional = true }
 pin-project-lite = "0.2.13"
 serde = { version = "1.0", features = ["derive"] }
+serde-wasm-bindgen = { version = "0.5.0", optional = true }
 thiserror = "1.0"
+rusty-leveldb = { git = "https://github.com/NumberFour8/leveldb-rs", branch = "feature/no-fs", default-features = false }
```

Comment:
This should either be forked to `hoprnet` or PRed and merged into the upstream leveldb-rs.